godtools/App/Share/Domain/UseCases/GetAllToolsUseCase/GetAllToolsUseCase.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  GetAllToolsUseCase.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 8/24/22.
+//  Copyright © 2022 Cru. All rights reserved.
+//
+
+import Combine
+
+class GetAllToolsUseCase {
+    
+    private let getToolUseCase: GetToolUseCase
+    private let resourcesRepository: ResourcesRepository
+    
+    init(getToolUseCase: GetToolUseCase, resourcesRepository: ResourcesRepository) {
+        self.getToolUseCase = getToolUseCase
+        self.resourcesRepository = resourcesRepository
+    }
+    
+    func getAllToolsPublisher(sorted: Bool) -> AnyPublisher<[ToolDomainModel], Never> {
```

Comment:
Hey @rachaelblue for the getAllTools won't those always be sorted by the default order?  Not sure this argument would be needed and may cause confusion.

Reply:
The only case is when we dynamically create the category buttons-- we get allTools, but sort them by language availability first and default order second, and then pull out the categories.  I could make this a default parameter like `func getAllToolsPublisher(sorted: Bool = true)` so we otherwise don't have to think about it?

Reply:
Hey @rachaelblue I forgot about sorting on the language availability. Lets leave this as is.